gto/api.py

```diff
@@ -87,6 +87,7 @@ def remove(repo: Union[str, Repo], name: str):
     return init_index_manager(path=repo).remove(name)
 
 
+@auto_push_on_remote_repo
```

Comment:
Not sure what caused this, but
```
$ gto register lol -r https://github.com/aguschin/example-gto
Created git tag 'lol@v0.0.1' that registers version
Running `git push origin lol@v0.0.1`
❌ The command `git push origin lol@v0.0.1` failed. Make sure your local repository is in sync with the remote.
```

Reply:
Mmm :( this is what happens when you use mocks so much and you don't know enough the library you are mocking.

Anyway, I pushed a fix on how I use gitpython https://github.com/iterative/gto/pull/286/commits/be4e34325516dd5a12e3c47e65244eeb63acd6d2

To me it seems to solve the issue, please double check

Reply:
Checked, works! Thanks!